ipfs-api/Cargo.toml

```diff
@@ -13,13 +13,20 @@ license                   = "MIT OR Apache-2.0"
 [badges]
 travis-ci                 = { repository = "ferristseng/rust-ipfs-api" }
 
+[features]
+default                   = ["hyper", "hyper-multipart-rfc7578"]
+actix                     = ["actix-web", "actix-multipart-rfc7578"]
+
 [dependencies]
+actix-multipart-rfc7578   = { version = "0.1", optional = true }
+actix-web                 = { version = "0.7", optional = true }
+common-multipart-rfc7578  = { version = "0.1.1" }
```

Comment:
I had set 0.1.1 because that includes `content_type()`, but `actix-multipart-rfc7578` and `hyper-multipart-rfc7578` specify 0.1.0. Will it automatically upgrade the deps to 0.1.1?

Reply:
I think it should if you do `cargo update`. Since I haven't published a new version yet, it should be safe to remove.